cmd/replica-estimator/app/options/options.go

```diff
@@ -0,0 +1,48 @@
+package options
+
+import (
+	"github.com/spf13/pflag"
+)
+
+const (
+	defaultBindAddress = "0.0.0.0"
+	defaultServerPort  = 10352
+	defaultHealthzPort = 10351
+)
+
+// Options contains everything necessary to create and run replica-estimator.
+type Options struct {
+	KubeConfig  string
+	Master      string
+	ClusterName string
+	// BindAddress is the IP address on which to listen for the --secure-port port.
+	BindAddress string
+	// SecurePort is the port that the server serves at.
+	SecurePort int
+	// ServerPort is the port that the server gRPC serves at.
+	ServerPort int
+	// ClusterAPIQPS is the QPS to use while talking with cluster kube-apiserver.
+	ClusterAPIQPS float32
+	// ClusterAPIBurst is the burst to allow while talking with cluster kube-apiserver.
+	ClusterAPIBurst int
+}
+
+// NewOptions builds an empty options.
+func NewOptions() *Options {
+	return &Options{}
+}
+
+// AddFlags adds flags of estimator to the specified FlagSet
+func (o *Options) AddFlags(fs *pflag.FlagSet) {
+	if o == nil {
+		return
+	}
+	fs.StringVar(&o.KubeConfig, "kubeconfig", o.KubeConfig, "Path to a KubeConfig. Only required if out-of-cluster.")
```

Comment:
It's the kubeconfig of karmada-apiserver? 
I suggest keeping consistent with the `karmada-agent`:
```
--karmada-kubeconfig string                     Path to karmada control plane kubeconfig file.
```

Reply:
And reservers the `--kubeconfig` to member cluster.

Reply:
OK.

Reply:
Well, I realize this config is for the member cluster. Actually, according to the desgin, this module does not have any connections with karmada-apiserver. It start a pod and node informer of member cluster and wait for scheduler to request max replicas.

Reply:
OK, then a `--kubeconfig` would be enough.